test/common/testobjects.go

```diff
@@ -88,18 +87,18 @@ func NewTestTargetObject(typeConfig typeconfig.Interface, namespace string, fixt
 	}
 
 	federate.SetBasicMetaFields(obj, typeConfig.GetTarget(), "", namespace, "test-e2e-")
+	if typeConfig.GetTarget().Kind == util.NamespaceKind {
+		// Federated namespace testing needs to have the same name as its namespace.
```

Comment:
Is this function returning a non-federated or federated resource?  The function name suggests the former, but this comment suggests the latter.

Reply:
It returns a non-federated resource. The comment should have been `Namespace crud testing...`. Updated!